packages/core-data/src/resolvers.js

```diff
@@ -25,15 +25,37 @@ import {
 import { getKindEntities, DEFAULT_ENTITY_KEY } from './entities';
 import { apiFetch, select, resolveSelect } from './controls';
 import { ifNotResolved, getNormalizedCommaSeparable } from './utils';
+import getQueryParts from './queried-data/get-query-parts';
 
 /**
  * Requests authors from the REST API.
+ *
+ * @param {Object} query Query object.
+ */
+export function* getAuthors( query = {} ) {
+	const path = addQueryArgs(
+		'/wp/v2/users/?who=authors&per_page=100',
```

Comment:
Should we remove the per_page argument here? the query can override it if I'm not wrong.

Reply:
Also it seems that the "who" argument shouldn't be hardcoded like that but merged into "query" in order for the "users per query" state to be populated properly.

It seems like right now, if one does `getUsers` with a similar query object, you might get bad results.

Reply:
Thinking more potentially we can remove this `getAuthors` selector/resolver as it's just a shortcut for `getUsers` AFAIK.

Reply:
I'll give `getUsers` a try.